### PR TITLE
Upgrade pax stable python version

### PR DESCRIPTION
**Pull request by gkroiz**
# Description
The pax stable tests include optax as a dependency, which on 1/15 released a new stable version dependent on a newer python version. As a result, this PR updates the python version for the pax stable tests.

# Tests
[screenshot](http://shortn/_naataIzJtM) of using new praxis/paxml 1.3.1 on stable e2e tests (all passed)

# Checklist
Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run one-shot tests and provided workload links above if applicable. 
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Review by RissyRan — APPROVED** [review]

**Review by RissyRan — CHANGES_REQUESTED** [review]
Waiting for test link. Thanks!

**Comment by gkroiz** [comment]
The fix to the stable tests might require a new pax version, in which case I will hold this PR off for the time being.

**Comment by gkroiz** [comment]
> The fix to the stable tests might require a new pax version, in which case I will hold this PR off for the time being.

new pax/praxis 1.3.1 is released, will open this PR, ready for review/merge

File: dags/solutions_team/configs/pax/solutionsteam_pax_supported_config.py [hunk 1]
```diff
@@ -77,9 +77,7 @@ def get_runtime_version(pax_version: PaxVersion, tpu_version: TpuVersion) -> str
   elif tpu_version is TpuVersion.V5P:
     return RuntimeVersion.V2_ALPHA_TPUV5.value
   else:
-    if pax_version is PaxVersion.STABLE:
-      return RuntimeVersion.TPU_VM_V4_BASE.value
-    elif pax_version is PaxVersion.NIGHTLY:
+    if pax_version in [PaxVersion.STABLE, PaxVersion.NIGHTLY]:
```
Comment (RissyRan) [inline dags/solutions_team/configs/pax/solutionsteam_pax_supported_config.py:80]: Do you think it's reasonable to set `RuntimeVersion.TPU_UBUNTU2204_BASE.value` as default for rest of TPU versions? As it works for both stable and nightly

**Review by RissyRan — APPROVED** [review]

**Pull request merged**